db/fixtures/miq_product_features.yml

```
@@ -4612,103 +4612,117 @@
-        :description: Remove Provier
-        :feature_type: admin
-        :identifier: provider_foreman_delete_provider
-      - :name: Edit Provider
-        :description: Edit a Provider
-        :feature_type: admin
-        :identifier: provider_foreman_edit_provider
-      - :name: Pause
-        :description: Pause a Provider
-        :feature_type: admin
-        :identifier: provider_foreman_pause
-      - :name: Resume
-        :description: Resume a Provider
-        :feature_type: admin
-        :identifier: provider_foreman_resume
-      - :name: Add Provider
-        :description: Add a Provider
-        :feature_type: admin
-        :identifier: provider_foreman_add_provider
-  - :name: Configuration Profiles
-    :description: Configuration Profiles
-    :feature_type: node
-    :identifier: configuration_profile
+      :identifier: ems_configuration_delete_provider
+    - :name: Edit Configuration Manager Provider
+      :description: Edit a Configuration Manager Provider
+      :feature_type: admin
+      :identifier: ems_configuration_edit_provider
+    - :name: Add Configuration Manager Provider
+      :description: Add a Configuration Manager Provider
+      :feature_type: admin
+      :identifier: ems_configuration_add_provider
+- :name: Configuration Profiles
+  :description: Everything under Configuration Profiles
+  :feature_type: node
+  :identifier: configuration_profile
+  :children:
+  - :name: View
+    :description: View Configuration Profiles
+    :feature_type: view
+    :identifier: configuration_profile_view
     :children:
-    - :name: View
-      :description: View Configuration Profiles
+    - :name: List
+      :description: Display Lists of Configuration Profiles
       :feature_type: view
-      :identifier: configuration_profile_view
-    - :name: Operate
-      :description: Perform Operations on Configuration Profiles
+      :identifier: configuration_profile_show_list
+    - :name: Show
+      :description: Display Individual Configuration Profile
+      :feature_type: view
+      :identifier: configuration_profile_show
+  - :name: Operate
+    :description: Perform Operations on Configuration Profiles
+    :feature_type: control
+    :identifier: configuration_profile_control
+    :children:
+    - :name: Edit Tags
+      :description: Edit Configuration Profiles Tags
       :feature_type: control
-      :identifier: configuration_profile_control
-      :children:
-      - :name: Edit Tags
-        :description: Edit Configuration Profiles Tags
-        :feature_type: control
-        :identifier: configuration_profile_tag
-  - :name: Configured Systems
-    :description: Everything under Configured Systems accordion
-    :feature_type: node
-    :identifier: configured_systems_filter_accord
+      :identifier: configuration_profile_tag
+- :name: Configured Systems
+  :description: Everything under Configured Systems
+  :feature_type: node
+  :identifier: configured_system
```

Singular here, plural below. It should be probably:
```suggestion
  :identifier: configured_systems
```